### PR TITLE
Cherrypick changes from (still unmerged sha256sum branch)

### LINKED ISSUE #563: Error downloading from URL for hmmer
Hello,

I'm trying to install from Galaxy's Tool shed repository, the 'package_hmmer_3_1b2' tool dependencies.

However, I'm getting this error:
Error downloading from URL
ftp://selab.janelia.org/pub/software/hmmer3/3.1b2/hmmer-3.1b2.tar.gz:
<urlopen error ftp error: [Errno 110] Connection timed out>

Can someone please help me?

Salutations!

### PR DESCRIPTION
**Pull request by erasche**
Fixes #563

*coughs*. We really need to make a decision on https://github.com/galaxyproject/tools-iuc/pull/213

### CODE REVIEW

**Comment by nsoranzo** [comment]
We should probably also `s|http://hmmer.janelia.org/|http://hmmer.org/|` in the 2 `tool_dependencies.xml` and in the 2 `.shed.yml` files.

**Comment by erasche** [comment]
Good suggestions @nsoranzo

File: packages/package_hmmer_3_1b2/tool_dependencies.xml [hunk 1]
```diff
@@ -3,7 +3,9 @@
     <package name="hmmer" version="3.1b2">
         <install version="1.0">
             <actions>
-                <action type="download_by_url">ftp://selab.janelia.org/pub/software/hmmer3/3.1b2/hmmer-3.1b2.tar.gz</action>
+                <action type="download_by_url" sha256sum="246f5e32545d3a5706d065d8413fc8fc5445e779a97019e98ba319e5ef9dc1d8">
+                    https://depot.galaxyproject.org/software/hmmer/hmmer_3.1b1_src_all.tar.gz
```
Comment (nsoranzo) [inline packages/package_hmmer_3_1b2/tool_dependencies.xml]: s/b1/b2

Comment (erasche) [inline packages/package_hmmer_3_1b2/tool_dependencies.xml]: odd, that shouldn't've happened.

Comment (nsoranzo) [inline packages/package_hmmer_3_1b2/tool_dependencies.xml]: Also the sha256sum is probably the wrong one.

Comment (erasche) [inline packages/package_hmmer_3_1b2/tool_dependencies.xml]: Yep, corrected as part of 9ccc550

**Pull request merged**